students/JamesTraub/Lesson02/JT-Fizz-Buzz.py

```diff
@@ -0,0 +1,14 @@
+# Write a program that prints the numbers from 1 to 100 inclusive.
+# But for multiples of three print “Fizz” instead of the number.
+# For the multiples of five print “Buzz” instead of the number.
+# For numbers which are multiples of both three and five print “FizzBuzz” instead.
+
+for number in range(1, 101):
+    if number % 3 == 0 and number % 5 == 0:
+        print('FizzBuzz')
+    elif number % 5 == 0:
+        print('Buzz')
+    elif number % 3 == 0:
+        print('Fizz')
+    else:
+        print(number)
```

Comment:
your file name should be `jt_fizz_buzz.py` 

for reference: https://www.python.org/dev/peps/pep-0008/#package-and-module-names

otherwise fizz buzz looks great!